axelor-human-resource/src/main/java/com/axelor/apps/hr/service/lunch/voucher/LunchVoucherMgtServiceImpl.java

```diff
@@ -245,62 +245,46 @@ public int updateStock(
 
   @Transactional(rollbackOn = {Exception.class})
   public void export(LunchVoucherMgt lunchVoucherMgt) throws IOException {
```

Comment:
Please create a new service LunchVoucherExportService and move this method there.